src/lib/eZ/RichText/Resources/stylesheets/docbook/xhtml5/edit/core.xsl

```diff
@@ -657,6 +657,12 @@
     <xsl:apply-templates select="node()|@*"/>
   </xsl:template>
 
+  <xsl:template match="docbook:eztemplate[@type='style']/docbook:ezcontent/text()">
```

Comment:
`@type` is deprecated, we shouldn't rely on this. If the problem exists it probably applies both to Styles and Tags.

Reply:
>   If the problem exists it probably applies both to Styles and Tags.

Actually, the problem applies only to Custom Styles.  The content of the Custom Tags is one or more block elements (paragraphs, ordered lists, etc) and the line breaks are transformed during processing of those block elements. Nevertheless, fixed in https://github.com/ezsystems/ezplatform-richtext/pull/34/commits/f7b6b1c50eb9afbc9c6d05c83d9637786ce40e9c  😉